README.md

```diff
@@ -1 +1,139 @@
-# OOP-School-Library
\ No newline at end of file
+<a name="readme-top"></a>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 OOP-School-Library](#-OOP-School-Library)
+  - [🛠 Built With ](#-built-with-)
+  - [🚀 Live Demo ](#-live-demo-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 OPP-School-Library<a name="about-project"></a>
+
+** OPP-School-Library**
+In this project, I build a school library app. This app will allow you
+
+- Add new students or teachers.
+- Add new books.
+- Save records of who borrowed a given book and when.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Server</summary>
+
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[Add new books]**
+- **[Add students and teachers]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+-[COMING SOON]()
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
```

Comment:
- [ ] [OPTIONAL]
Kindly ensure that the _Getting Started_ section_ is filled with the appropriate instructions on how to _setup_ and _run_ your app. Implementing the above change(s) will make your ReadMe/ repository stand out even better! 😉